e2e/spec.cy.ts

```diff
@@ -0,0 +1,5 @@
+describe('empty spec', () => {
+  it('passes', () => {
+    cy.visit('https://example.cypress.io')
+  })
+})
```

Comment:
duplicate file & folder

Reply:
Sorry, I didn't understand. Where to duplicate?

Reply:
![2022-10-13_13h26_13](https://user-images.githubusercontent.com/58398016/195572905-1c08029d-e059-4704-8fb4-d4199d4ca2cf.png)